client/src/styles/passwordchange.scss

```diff
@@ -0,0 +1,23 @@
+.app-login-form {
+  max-width: 380px;
```

Comment:
### Nitpick

This looks like one-space indentation, it should either be 2-space indentation similar to your JS files or 4-spaces.